src/components/Sum/Sum.jsx

```diff
@@ -1 +1,13 @@
-// export const Sum = () => ();
+/* eslint-disable react/jsx-one-expression-per-line */
+import React from 'react';
+
+export const Sum = ({
+  a = 0,
+  b = 0,
+}) => {
+  const sum = a + b;
+
+  return (
+    <p>Sum of {a} and {b} is {sum}</p>
```

Comment:
The rendered string could be improved for better readability. Consider using template literals to interpolate the variables in the string.